newIDE/app/src/MainFrame/EditorContainers/HomePage/CardWidget.js

```diff
@@ -0,0 +1,70 @@
+// @flow
+import * as React from 'react';
+import { createStyles, makeStyles, Paper } from '@material-ui/core';
```

Comment:
createStyles and makeStyles can be imported from /core/styles and Paper from /core/Paper